src/components/StoryCards/StoryCard.tsx

```diff
@@ -13,9 +13,11 @@ import styles from './StoryCard.module.scss';
 
 type Props = {
     className?: string;
+    fallback: StoryImage.Props['fallback'];
     forceAspectRatio?: boolean;
-    isExternal: boolean;
+    isExternal: ExternalStoryUrl;
```

Comment:
It's a bit weird to see this boolean-like `is-` property used for data. 

As a suggestion, what do you think about naming it just `external` (without the `is-` prefix)?

Reply:
Sure, sounds good to me, I'll change it.